subsys/bluetooth/controller/ll_sw/nordic/lll/lll_conn.c

```diff
@@ -156,6 +170,17 @@ void lll_conn_abort_cb(struct lll_prepare_param *prepare_param, void *param)
 
 	/* NOTE: This is not a prepare being cancelled */
 	if (!prepare_param) {
+		/* Get reference to LLL connection context */
+		lll = param;
+
+		/* For a peripheral role, ensure at least one PDU is tx-ed
+		 * back to central, otherwise let the supervision timeout
+		 * countdown be started.
+		 */
+		if (lll->role && (trx_cnt <= 1U)) {
```

Comment:
If `role` is a specific value so that role == 0 == central and role == 1 == peripheral, then please make the role check more explicit. 

```suggestion
		if (lll->role == BT_HCI_ROLE_PERIPHERAL && (trx_cnt <= 1U)) {
```

The above assumes that the `role` field uses the HCI roles